pages/open_cas_linux/guide_terminology.md

```diff
@@ -0,0 +1,33 @@
+| guest              | An operating system running on a Virtual Machine (VM) environment.                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| host               | The operating system running (hosting) the Virtual Machine (VM) environment, on which guest Operating Systems can be installed.                                                                                                                                                                                                                                                                                                                                                                                                              |
+| hypervisor         | A hardware virtualization technique that allows multiple operating systems, termed guests, to run concurrently on a host computer.                                                                                                                                                                                                                                                                                                                                                                                                           |
+| IO                 | Abbreviation for input/output as it relates to the flow of data.                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| lazy write         | The process of mirroring to primary storage. See also write-back.                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
The 'lazy write' is used in this documentation once, in description of WB, to which we redirect here...